src/components/Badge/RangeBadge.tsx

```diff
@@ -1,5 +1,3 @@
-import React from 'react'
```

Comment:
I'm seeing this in several files, I'll stop adding comments regarding it.
Is this no longer required for tsx files?

Reply:
Maybe not, since 1 year ago https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Was our version that old? :thinking: